drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -1298,6 +1298,11 @@ void RasterizerSceneGLES3::_fill_render_list(RenderListType p_render_list, const
 					RID light = light_storage->light_instance_get_base_light(light_instance);
 					int32_t shadow_id = light_storage->light_instance_get_shadow_id(light_instance);
 
+					//Skip lights that do not contribute to the current instance
+					if (!(light_storage->light_get_cull_mask(light) & inst->layer_mask)) {
```

Comment:
If the light is already culled in `_scene_cull()` then this check is unnecessary right?

Reply:
Yes. I skipped the check and it works just fine. (sry for that!)